apps/client/src/components/globalSettings/UnivariateCellPlot.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+    <div v-if="cellMetaData.dataInitialized">
+        <q-item-section style="position: relative">
+            <div
+                id="plotContainer"
+                ref="vgPlotContainer"
+                @clearSelection="clearBrushSelection"
+                style="position: relative"
+            ></div>
+            <div
+                class="selection-box left"
+                style="position: absolute; bottom: 12px"
+            >
+                <input
```

Comment:
I do like the styles you have come up with for these, but it might be easier to use the q-input component with type=number. This gets you a number of things for free (user can only type numbers, the variable is converted to a number for you)